nix/overlay.nix

```diff
@@ -87,7 +87,7 @@ self: super: {
     inherit (super) stdenv fetchurl;
   };
 
-  helm = super.callPackage ./pkgs/helm {};
+  helm = super.callPackage ./pkgs/helm { };
```

Comment:
This is not a semantic change, or is it?

Reply:
It's not, but `treefmt` put it there.